.github/workflows/versioning.yml

```
@@ -0,0 +1,14 @@
+---
+name: Tag Versioning
+
+on:
+  release:
+    types: 
```

Looks like trailing whitespace on this line is making `pre-commit` sad.

```suggestion
    types:
```